src/python/pants/build_graph/register.py

```diff
@@ -37,19 +25,6 @@ def __call__(self):
 
 def build_file_aliases():
     return BuildFileAliases(
-        targets={
-            "alias": AliasTargetFactory(),
-            "files": Files,
-            "prep_command": PrepCommand,
-            "resources": Resources,
-            "remote_sources": RemoteSources,
-            "target": Target,
-        },
-        objects={"get_buildroot": get_buildroot, "netrc": Netrc, "pants_version": pants_version},
-        context_aware_object_factories={
-            "buildfile_path": BuildFilePath,
-            "intransitive": IntransitiveDependencyFactory,
-            "provided": ProvidedDependencyFactory,
-            "scoped": ScopedDependencyFactory,
-        },
+        objects={"get_buildroot": get_buildroot, "pants_version": pants_version},
+        context_aware_object_factories={"buildfile_path": BuildFilePath},
```

Comment:
I'm not sure if there's any reason to keep these. They seem possibly helpful and they work just fine in v2.